java/system-test/system-test-data-generation/src/main/java/sleeper/systemtest/datageneration/IngestRandomDataToDocker.java

```diff
@@ -0,0 +1,112 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sleeper.systemtest.datageneration;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.s3.S3AsyncClient;
+import software.amazon.awssdk.services.s3.S3AsyncClientBuilder;
+
+import sleeper.configuration.jars.ObjectFactory;
+import sleeper.configuration.jars.ObjectFactoryException;
+import sleeper.configuration.properties.instance.InstanceProperties;
+import sleeper.configuration.properties.table.TableProperties;
+import sleeper.ingest.IngestFactory;
+import sleeper.statestore.StateStoreProvider;
+import sleeper.systemtest.configuration.SystemTestProperties;
+import sleeper.utils.HadoopConfigurationProvider;
+
+import java.io.IOException;
+import java.net.URI;
+
+import static sleeper.configuration.utils.AwsV1ClientHelper.buildAwsV1Client;
+import static sleeper.systemtest.configuration.SystemTestProperty.NUMBER_OF_RECORDS_PER_WRITER;
+
+public class IngestRandomDataToDocker {
+    private final InstanceProperties instanceProperties;
+    private final TableProperties tableProperties;
+    private final AmazonDynamoDB dynamoDB;
+    private final long numberOfRecords;
+
+    public IngestRandomDataToDocker(InstanceProperties instanceProperties, TableProperties tableProperties,
+                                    AmazonDynamoDB dynamoDB, long numberOfRecords) {
+        this.instanceProperties = instanceProperties;
+        this.tableProperties = tableProperties;
+        this.dynamoDB = dynamoDB;
+        this.numberOfRecords = numberOfRecords;
+    }
+
+    private void run() throws IOException {
+        SystemTestProperties systemTestProperties = new SystemTestProperties();
+        systemTestProperties.setNumber(NUMBER_OF_RECORDS_PER_WRITER, numberOfRecords);
+        StateStoreProvider stateStoreProvider = new StateStoreProvider(dynamoDB,
+                instanceProperties, HadoopConfigurationProvider.getConfigurationForECS(instanceProperties));
+        WriteRandomDataDirect.writeWithIngestFactory(
+                IngestFactory.builder()
+                        .objectFactory(ObjectFactory.noUserJars())
+                        .localDir("/tmp/scratch")
+                        .stateStoreProvider(stateStoreProvider)
+                        .s3AsyncClient(buildS3AsyncClient(S3AsyncClient.builder()))
+                        .instanceProperties(instanceProperties)
+                        .build(),
+                systemTestProperties, tableProperties);
```

Comment:
I thought this was going to write a Parquet file locally rather than running an ingest.

Reply:
We could have one script to do ingest and one just to write a file locally, eg. `ingestRandomData.sh` and `generateRandomData.sh`.